src/script/squirrel.hpp

```diff
@@ -21,6 +21,12 @@ enum class ScriptType : uint8_t {
 
 struct ScriptAllocator;
 
+struct Tag {
+	std::string tag;
+	auto operator<=>(const Tag &) const = default;
+};
+inline auto operator<=>(const std::unique_ptr<Tag> &a, const std::unique_ptr<Tag> &b) { return *a.get() <=> *b.get(); };
```

Comment:
I think `.get()` isn't needed here, you can dereference `std::unique_ptr` directly.